source/connect/advanced-connection-options/advanced-connection.txt

```diff
@@ -25,6 +25,10 @@ Procedure
 .. procedure::
    :style: normal
 
+   .. step:: Open the :guilabel:`New Connection` window.
```

Comment:
Are we referring to modal dialogs windows? Seems like that would be confusing given that window means something else already.

Reply:
I changed all mentions to use the word "modal"